packages/dashboard/src/scss/main.scss

```diff
@@ -181,6 +188,40 @@
     }
   }
 
+  .collapsable-multiviz-container {
```

Comment:
Nit/No Fix Needed: Styles in this file look good. My primary concern with styles lately has been the location of the styles, ie. if they can be placed closer to the component instead of the main.scss file (maybe collapsibleVisualizationRow.styles.css). However, that isn't a pattern that's followed in this repo regularly so this is more of a nit. It's more about where other developers can find the styles easily.